plugins/integration/IntegrationProviderPlugin.php

```diff
@@ -34,6 +34,14 @@ public static function getEnums($baseEnumName = null)
 		return array();
 	}
 
+	/**
+	 * @return IIntegrationProvider
+	 */
+	public static function getProvider($providerType)

```

Comment:
The only purpose of this method is to enforce all implementations of IntegrationProviderPlugin to implement also an IIntegrationProvider.